site/content/en/docs/Installation/Creating Cluster/aks.md

```diff
@@ -65,9 +65,9 @@ az network nsg rule create \
   --destination-port-range 7000-8000
   ```
 
-### Creating and assigning Public IPs to Nodes
+### Getting Public IPs to Nodes
 
-Nodes in AKS don't get a Public IP by default. To assign a Public IP to a Node, find the Resource Group where the AKS resources are installed on the [portal](https://portal.azure.com) (it should have a name like `MC_resourceGroupName_AKSName_westeurope`). Then, you can follow the instructions [here](https://docs.microsoft.com/en-us/azure/site-recovery/concepts-public-ip-address-with-site-recovery) to create a new Public IP and assign it to the Node/VM. For more information on Public IPs for VM NICs, see [this document](https://docs.microsoft.com/azure/virtual-network/virtual-network-network-interface-addresses). If you are looking for an automated way to create and assign Public IPs for your AKS Nodes, check [this project](https://github.com/dgkanatsios/AksNodePublicIPController).
+Resource Group where the AKS resources are installed on the [portal](https://portal.azure.com) are named like `MC_resourceGroupName_AKSName_westeurope`). Find `Virtual machine scale set` -> `Instances` -> `Public IP address`. For more information on Public IPs for VM NICs, see [this document](https://docs.microsoft.com/azure/virtual-network/virtual-network-network-interface-addresses). 
```

Comment:
For clarity, "instances" is only viewable after clicking a particular set within the `Virtual Machine Scale Sets` UI, and the public IP is only viewable after selecting the individual instance.

I suppose that's implied. But if explicitness is desired, the instructions should be more like this:

> To find a resource's public IP, search for `Virtual Machine Scale Sets` -> click on the set name -> click `Instances` -> click on the instance name -> view `Public IP address`

Reply:
Also, is the entire first sentence obsolete now? Considering that Azure now has `--enable-node-public-ip`, this section isn't really serving an explicit purpose aside from letting people know how to view the IP from the UI (don't need the resource group name for that).

Last thought... Should we mention here that the IP can be viewed via `kubectl get nodes`, to avoid any confusion leading people around the UI unnecessarily?

Reply:
will retain only one simples way to see IP from UI. so I do not think it is obsolete until https://github.com/Azure/AKS/issues/2318  .

Reply:
i tried to get nodes - azure does not implements that spec, see issues mentioned.

Reply:
is there a way to get the public IP listed via kubectl get gameservers? right now is picking up the private ip

Reply:
see linked issues above

Reply:
> will retain only one simples way to see IP from UI. so I do not think it is obsolete until [Azure/AKS#2318](https://github.com/Azure/AKS/issues/2318) .

Sorry, didn't mean to say the entire thing is obsolete. But maybe we don't need the first sentence, regarding the resource group?